cpptod.dd

```diff
@@ -829,6 +831,106 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="functioninputparameters">Function Input Parameters</a>)
```

Comment:
Hi @wilzbach, D is a incredibly powerful but easy to use language. It would be a shame if it would not be "promoted" accordingly... the *Programming in D for C/C++ Programmers* sections of the dlang.org were my starting point. It is fun to learn a new language and to get to know your current languages from a different perspective.

To be honest... I copy&pasted it from the above sections in the file to keep it consistent w.r.t. styling.